### DISCUSSION TITLE
Move taskbar from the default bottom position?

### DISCUSSION THREAD

**Discussion by tmaxmax** [valinet/ExplorerPatcher#435] [Q&A] [answered]
I'd like to move the taskbar to the right side of the screen on Windows 11. Can I achieve this using ExplorerPatcher?

**Comment by valinet** [comment, marked as answer]
> "taskbar"

Taskbar it's like saying you want to live someplace on Earth. Can you be more specific? Which taskbar? Windows 10 or Windows 11 one? The Windows 10 one can be moved to whatever side you want by right click on it - uncheck "Lock all taskbars" or "Lock taskbar", then dragging it towards whatever side of the screen, while the Windows 11 one is officially stuck to the bottom but with some registry tweaks it can also be moved to the top (use Winaero Tweaker for this, for example).

**Reply by tmaxmax** [reply]
I apologize for being vague. I really only wanted my taskbar to the right, regardless of whether it is the Windows 10 version or Windows 11 one. Thank you for the response, I feel ridiculous now as I have never known that I could move the taskbar so easily in Windows 10 too - I was always going to the settings!

**Reply by valinet** [reply]
Yeah, no worries, actually, apparently, quite a few people do not know about this, you're not the only one. There have been many more questions regarding this topic, with people not knowing this trick. Thanks.

**Reply by Anixx** [reply]
>  as I have never known that I could move the taskbar so easily in Windows 10 too

Did not you use Windows 95? This feature exists in all Windows versions since then.

**Reply by tmaxmax** [reply]
Using Windows since XP... well, maybe I knew once about it, but forgotten in the meantime.
